dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -74,6 +74,7 @@ RUN for iter in {1..10}; do microdnf update -y && microdnf install -y tar gzip f
 {{- end }}
 
 {{- if contains .from "ubuntu" }}
+RUN touch /var/mail/ubuntu && chown ubuntu /var/mail/ubuntu && userdel -r ubuntu
```

Comment:
Why do we need to create `/var/mail/ubuntu` and then delete the ubuntu user ?

Reply:
Hey @pchila - I believe the following is the justification for this line. 

https://bugs.launchpad.net/cloud-images/+bug/2005129/comments/2

Reply:
Added comment.